src/index.js

```diff
@@ -1,60 +1,92 @@
+import editTask from './modules/editTask.js';
+import deleteTask from './modules/deleteTask.js';
+import deleteCompleted from './modules/deleteCompleted.js';
 
 const container = document.querySelector('.container');
+const taskInput = document.querySelector('.italics');
+const addButton = document.querySelector('.add');
+const clearCompletedLink = document.querySelector('.clear-all');
 
-const tasksList = [
-  {
-    description: 'Do this',
-    completed: true,
-    index: 0,
-  },
-  {
-    description: 'Do that',
-    completed: false,
-    index: 1,
-  },
-];
-
-const renderTasks = (tasks) => {
-  for (let i = 0; i <= tasks.length - 1; i++) {
-    const taskContainer = document.createElement('div');
-    taskContainer.id = tasks[i].index;
-    taskContainer.classList.add('list');
-
-    const taskCheckox = document.createElement('input');
-    taskCheckox.type = 'checkbox';
-    taskCheckox.classList.add('check');
-
-    const taskDescription = document.createElement('label');
-    taskDescription.classList.add('label');
-    taskDescription.textContent = `${tasks[i].description}`;
-    taskDescription.contentEditable = true;
-
-    const dots = document.createElement('span');
-    dots.innerHTML = "<i class='fas fa-ellipsis-v'></i>";
-
-    const trash = document.createElement('span');
-    trash.innerHTML = "<i class='fas fa-trash-alt'></i>";
-    trash.style.display = 'none';
-
-    taskContainer.appendChild(taskCheckox);
-    taskContainer.appendChild(taskDescription);
-    taskContainer.appendChild(dots);
-    taskContainer.appendChild(trash);
-    container.appendChild(taskContainer);
-
-    taskDescription.addEventListener('focus', () => {
-      dots.style.display = 'none';
-      trash.style.display = 'flex';
-    });
-
-    taskDescription.addEventListener('blur', () => {
-      dots.style.display = 'flex';
-      trash.style.display = 'none';
-    });
+const renderTasks = () => {
+  while (container.lastChild) {
+    container.removeChild(container.lastChild);
+  }
+
+  const storedList = JSON.parse(localStorage.getItem('todo-list'));
+  if (storedList != null) {
+    for (let i = 0; i <= storedList.length - 1; i++) {
+      const taskContainer = document.createElement('div');
+      taskContainer.id = storedList[i].index;
+      taskContainer.classList.add('list');
+
+      const taskCheckbox = document.createElement('input');
+      taskCheckbox.type = 'checkbox';
+      taskCheckbox.classList.add('check');
+
+      const taskDescription = document.createElement('label');
+      taskDescription.classList.add('label');
+      taskDescription.textContent = `${storedList[i].description}`;
+      taskDescription.contentEditable = true;
+
+      if (storedList[i].completed) {
+        taskCheckbox.checked = true;
+        taskDescription.classList.add('strike');
+      }
+
+      const trash = document.createElement('span');
+      trash.innerHTML = "<i class='fas fa-trash-alt'></i>";
+      trash.style.cursor = 'pointer';
```

Comment:
- [ ] Kindly inplement the delete feature on the `delete` button for each task as is in the provided [video](https://www.youtube.com/watch?v=AcUd-_Yjjqg)